prometheus_es_exporter/__init__.py

```diff
@@ -392,9 +393,11 @@ def conv(value):
               help='Detail level to log. (default: INFO)')
 @click.option('--verbose', '-v', default=False, is_flag=True,
               help='Turn on verbose (DEBUG) logging. Overrides --log-level.')
+@click_config_file.configuration_option()
 def cli(**options):
     """Export Elasticsearch query results to Prometheus."""
-
+    print(options)
+    return
```

Comment:
I presume this is left over from testing?

Reply:
Yeah. I've removed it